data/mods/Aftershock/items/crafting_scrap/biomaterial_scrap.json

```diff
@@ -28,7 +28,7 @@
     "id": "afs_biomaterial_4",
     "copy-from": "afs_scrap_4",
     "name": { "str_sp": "self healing polymers" },
-    "description": "Materials capable of taking environmental chemicals and utilizing them for self-repair; be careful you don't make grey goo.",
```

Comment:
For Aftershock since it’s a multicultural future I personally would prefer multiple spellings being acceptable on descriptions. I’ll confirm with the other Aftershock maintainers.

Reply:
That would make sense for alternative regions/mods indeed; whatever is decided, it's best to make it clear in manual of style.

Reply:
The other aftershockers have convinced me I’m being cruel to translators. We’ll do standardized US English spelling except where commented out for exact spellings.